SharedExpensesApp/app/src/main/java/com/example/sharedexpensesapp/ui/screens/BalanceViewModel.kt

```diff
@@ -0,0 +1,45 @@
+package com.example.sharedexpensesapp.ui.screens
```

Comment:
Since we have already many files in that package, I would move ViewModels to a new package